ProcessLib/BoundaryCondition/ConstraintDirichletBoundaryCondition.cpp

```diff
@@ -219,7 +219,8 @@ void ConstraintDirichletBoundaryCondition::getEssentialBCValues(
     bc_values.ids.emplace_back(current_id);
     bc_values.values.emplace_back(sum / cnt);
 
-    DBUG("Found %d dirichlet values.", bc_values.ids.size());
+    DBUG("Found %d constraint dirichlet boundary condition values.",
+         bc_values.ids.size());
     for (unsigned i = 0; i < bc_values.ids.size(); ++i)
     {
         DBUG("\tid: %d, value: %e", bc_values.ids[i], bc_values.values[i]);
```

Comment:
This and the above DBUG messages are quite noisy...

Reply:
Removed the for loop. The debug message above (lines 222/223) is only outputted once per application of the constraint dirichlet bc. The output can be helpful if there are issues setting the boundary condition. I would keep this debug message.